packages/cli/turbo.json

```diff
@@ -1,6 +1,9 @@
 {
 	"$schema": "http://turbo.build/schema.json",
 	"extends": ["//"],
-
-	"pipeline": {}
+	"tasks": {
+		"build": {
+			"outputs": ["dist/**"]
+		}
```

Comment:
Does overriding this here also override the `dependsOn` field at the top level?
Do we need to include that?
I see that this package depends upon the eslint and tsconfig packages, so we need those deps marked in turbo so that we don't fail to rebuild this package when those change.

Reply:
I guess it improves the upstream tasks that rely upon this task?